client/src/partials/templates/bhAccountSelect.tmpl.html

```diff
@@ -0,0 +1,29 @@
+<ng-form name="{{ $ctrl.name }}" data-bh-account-select>
+  <div class="form-group" ng-class="{ 'has-error' : $ctrl.validationTrigger && AccountForm.account.$invalid }">
+    <label class="control-label">
+      {{ $ctrl.label | translate }}
+    </label>
+
+    <ui-select
+     name="account"
+     ng-model="$ctrl.initialValue"
+     ng-model-options="{ 'debounce' : 200 }"
```

Comment:
This currently uses a `debounce` which seems to cause issues described https://github.com/IMA-WorldHealth/bhima-2.X/issues/972. We should address this bug before allowing this everywhere in master.